hedera-node/hedera-app/src/main/java/com/hedera/node/app/workflows/purechecks/PureChecksContextImpl.java

```diff
@@ -70,4 +71,15 @@ public TransactionBody body() {
     public Configuration configuration() {
         return configuration;
     }
+
+    @NonNull
+    @Override
+    public void executeInnerPureCheck(@NonNull TransactionBody body) throws PreCheckException {}
```

Comment:
You could probably follow the same pattern as `bodyParser` for using dispatcher also. I don't see why we need to inject dispatcher.